packages/css/documentation/coding-conventions.md

```diff
@@ -35,3 +35,11 @@ Still, we define its thickness and offset for the initial state.
   }
 }
 ```
+
+## Use mixins for patterns and resets
+
+We use Sass mixins to extract common patterns, especially if they need more than 1 declaration.
+We collect reset styles in mixins as well.
+Both the name of the mixins and their documentation help explain the approach.
+Mixins sit at the end of the declaration block to ensure their effect.
+The general ‘reset’ mixin is the very last.
```

Comment:
I’m also used to putting the reset first, but somehow they ended up being last in our code. Made me think about it and concluded that last may be better to prevent styles overriding the reset. The reset should always win.

Reply:
> The reset should always win.

Should it? It kind of prevents us from making generic resets. Say we make a generic button reset, with `background-color: transparent` to get rid of the grey background color. We would be unable to override that color in a component. 

(Again, not really an issue with our current resets, so we could leave it for now.)

Reply:
I didn’t think hard enough, apparently! Removed these as well. What a crazy PR this has become.

Reply:
Touching basic building blocks is always tricky :D I do like the idea of describing how our resets work and where you should define them though, do you want to do that in a separate ticket?

Reply:
Yes! Please! 😝